tests/blocks/test_checksum_consistency.py

```diff
@@ -5,20 +5,21 @@
 
 Checksums have been generated using prefect release 2.0.2.
 """
-
 import json
 from pathlib import Path
 
+import pytest
+
 from prefect.blocks.core import Block
 from prefect.utilities.dispatch import get_registry_for_type
 
+with open(str(Path(__file__).parent) + "/checksums.json", "r") as f:
+    standard_checksum = json.load(f)
+block_registry = get_registry_for_type(Block) or {}
 
-def test_checksums_are_consistent():
-    with open(str(Path(__file__).parent) + "/checksums.json", "r") as f:
-        standard_checksum = json.load(f)
 
-    for block_name, block in get_registry_for_type(Block).items():
-        assert (
-            block_name not in standard_checksum.keys()
-            or block._calculate_schema_checksum() == standard_checksum[block_name]
-        )
+@pytest.mark.parametrize("block_key,block", block_registry.items())
```

Comment:
I think you're okay here, but note that you need to be careful passing dynamic values as parameters. If the order of the items changes per execution of this file, the parallel test workers will fail. Usually an issue with `set` types.